packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/Series/BasicSeriesConfiguration.tsx

```diff
@@ -20,19 +20,12 @@ type BasicSeriesConfigurationProps = {
     item: Field | TableCalculation | CustomDimension;
     updateSingleSeries: (series: Series) => void;
     dragHandleProps?: DraggableProvidedDragHandleProps | null;
-
-    /**
-     * Temporary - we need to keep track of the series' index to assign a fallback color
-     * if shared colors are not enabled.
-     */
-    seriesIndex: number;
 };
 
 const BasicSeriesConfiguration: FC<BasicSeriesConfigurationProps> = ({
     isSingle,
     layout,
     series,
-    seriesIndex,
```

Comment:
Since series fallback colors are back to being pre-calculated (like they were before the color assignment changes), we no longer need the series index at all within child components.